cleaner/cleaner.go

```diff
@@ -0,0 +1,38 @@
+package cleaner
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+type t func() error
+
+var cleaners []t
+var inactive bool
+
+// Register a cleaner function. When a function is registered, the Signal watcher is started in a goroutine.
+func Register(f t) {
+	cleaners = append(cleaners, f)
+	if !inactive {
+		inactive = false
+		go func() {
+			ch := make(chan os.Signal, 1)
+			signal.Notify(ch, syscall.SIGINT, syscall.SIGTERM, os.Interrupt)
+			<-ch
+			fmt.Println("")
```

Comment:
I suppose it's to have a clean line in the shell ? 
```suggestion
                        // Prevent un-terminated ^C character in terminal
			fmt.Println()
```

Reply:
Precisely !